### PR TITLE
[ko] Update outdated files in dev-1.23-ko.1 (M38-M42)

### PR DESCRIPTION
**Pull request by jihoon-seo**
- Related issue: #30845 (**dev-1.23-ko.1**)
- Related PR: #30851 (**dev-1.22-ko.4**)

> * [ ]  M38.  `content/en/docs/concepts/workloads/controllers/ttlafterfinished.md`  | 22(+S) 29(-)
> * [ ]  M39.  `content/en/docs/concepts/workloads/pods/disruptions.md`  | 2(+XS) 2(-)
> * [ ]  M40.  `content/en/docs/concepts/workloads/pods/ephemeral-containers.md`  | 1(+XS) 10(-)
> * [ ]  M41.  `content/en/docs/concepts/workloads/pods/pod-lifecycle.md`  | 72(+M) 42(-)
> * [ ]  M42.  `content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md`  | 9(+XS) 5(-)

/language ko

### CODE REVIEW

File: content/ko/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 1]
```diff
@@ -233,57 +233,87 @@ kubelet은 파드의 [조건](#파드의-조건)을 `ContainerReady` 로 설정
...
+  명령어가 상태 코드 0으로 종료되면 진단이 성공한 것으로 간주한다.
 
-* [HTTPGetAction](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#httpgetaction-v1-core)은
-  지정한 포트 및 경로에서 컨테이너의 IP주소에
-  대한 HTTP `GET` 요청을 수행한다. 응답의 상태 코드가 200 이상 400 미만이면
+`grpc`
+: [gRPC](https://grpc.io/)를 사용하여 
+  원격 프로시저 호출을 수행한다. 
+  체크 대상이 [gRPC 헬스 체크](https://grpc.io/grpc/core/md_doc_health-checking.html)를 구현해야 한다. 
+  응답의 `status`가 `SERVING`이면
```
Comment (yoonian) [inline content/ko/docs/concepts/workloads/pods/pod-lifecycle.md:256]:
```suggestion
  응답의 `status` 가 `SERVING` 이면 
```

File: content/ko/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 2]
```diff
@@ -233,57 +233,87 @@ kubelet은 파드의 [조건](#파드의-조건)을 `ContainerReady` 로 설정
...
-* `startupProbe`: 컨테이너 내의 애플리케이션이 시작되었는지를 나타낸다.
-   스타트업 프로브(startup probe)가 주어진 경우, 성공할 때까지 다른 나머지 프로브는
-   활성화되지 않는다. 만약 스타트업 프로브가 실패하면, kubelet이 컨테이너를 죽이고,
-   컨테이너는 [재시작 정책](#restart-policy)에 따라 처리된다. 컨테이너에 스타트업
-   프로브가 없는 경우, 기본 상태는 `Success`이다.
+`livenessProbe`
+: 컨테이너가 동작 중인지 여부를 나타낸다. 만약
+  활성 프로브(liveness probe)에 실패한다면, kubelet은 컨테이너를 죽이고, 해당 컨테이너는
+  [재시작 정책](#restart-policy)의 대상이 된다. 만약 컨테이너가
+  활성 프로브를 제공하지 않는 경우, 기본 상태는 `Success`이다.
```
Comment (yoonian) [inline content/ko/docs/concepts/workloads/pods/pod-lifecycle.md:297]:
```suggestion
  활성 프로브를 제공하지 않는 경우, 기본 상태는 `Success` 이다.
```

File: content/ko/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 3]
```diff
@@ -233,57 +233,87 @@ kubelet은 파드의 [조건](#파드의-조건)을 `ContainerReady` 로 설정
...
+
+`readinessProbe`
+: 컨테이너가 요청을 처리할 준비가 되었는지 여부를 나타낸다.
+  만약 준비성 프로브(readiness probe)가 실패한다면, 엔드포인트 컨트롤러는
+  파드에 연관된 모든 서비스들의 엔드포인트에서 파드의 IP주소를 제거한다. 준비성 프로브의
+  초기 지연 이전의 기본 상태는 `Failure`이다. 만약 컨테이너가 준비성 프로브를
```
Comment (yoonian) [inline content/ko/docs/concepts/workloads/pods/pod-lifecycle.md:303]:
```suggestion
  초기 지연 이전의 기본 상태는 `Failure` 이다. 만약 컨테이너가 준비성 프로브를
```

File: content/ko/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 4]
```diff
@@ -233,57 +233,87 @@ kubelet은 파드의 [조건](#파드의-조건)을 `ContainerReady` 로 설정
...
+  지원하지 않는다면, 기본 상태는 `Success`이다.
```
Comment (yoonian) [inline content/ko/docs/concepts/workloads/pods/pod-lifecycle.md:304]:
```suggestion
  지원하지 않는다면, 기본 상태는 `Success` 이다.
```

File: content/ko/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 5]
```diff
@@ -233,57 +233,87 @@ kubelet은 파드의 [조건](#파드의-조건)을 `ContainerReady` 로 설정
...
+
+`startupProbe`
+: 컨테이너 내의 애플리케이션이 시작되었는지를 나타낸다.
+  스타트업 프로브(startup probe)가 주어진 경우, 성공할 때까지 다른 나머지 프로브는
+  활성화되지 않는다. 만약 스타트업 프로브가 실패하면, kubelet이 컨테이너를 죽이고,
+  컨테이너는 [재시작 정책](#restart-policy)에 따라 처리된다. 컨테이너에 스타트업
+  프로브가 없는 경우, 기본 상태는 `Success`이다.
```
Comment (yoonian) [inline content/ko/docs/concepts/workloads/pods/pod-lifecycle.md:311]:
```suggestion
  프로브가 없는 경우, 기본 상태는 `Success` 이다.
```

**Review by yoonian — COMMENTED** [review]
@jihoon-seo 확인해보았습니다.

**Comment by jihoon-seo** [comment]
@yoonian 리뷰 감사합니다. 리뷰 의견을 모두 반영하였습니다. 😊

**Pull request merged**